weed/operation/upload_content.go

```diff
@@ -5,6 +5,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/rclone/rclone/lib/encoder"
```

Comment:
I am concerned about the executable binary size and want to avoid rclone for normal build.